packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -1576,6 +1576,90 @@ module('Integration | operator-mode', function (hooks) {
       matrixService.sendMessage = originalSendMessage;
     });
 
+    test('displays specific error message for a message that is too large', async function (assert) {
+      await setCardInOperatorModeState(`${testRealmURL}Person/fadhlan`);
+      await renderComponent(
+        class TestDriver extends GlimmerComponent {
+          <template>
+            <OperatorMode @onClose={{noop}} />
+            <CardPrerender />
+          </template>
+        },
+      );
+
+      let originalSendMessage = matrixService.sendMessage;
+      let clientGeneratedId = '';
+      let event: any;
+      matrixService.sendMessage = async function (
+        roomId: string,
+        body: string,
+        _attachedCards: [],
+        _clientGeneratedId: string,
+        _context?: any,
+      ) {
+        clientGeneratedId = _clientGeneratedId;
+        event = {
+          event_id: 'test-event-id',
+          room_id: roomId,
+          state_key: 'state',
+          type: 'm.room.message',
+          sender: matrixService.userId!,
+          content: {
+            body,
+            msgtype: 'org.boxel.message',
+            formatted_body: body,
+            format: 'org.matrix.custom.html',
+            clientGeneratedId,
+          },
+          origin_server_ts: new Date(2024, 0, 3, 12, 30).getTime(),
+          unsigned: {
+            age: 105,
+            transaction_id: '1',
+          },
+          status: EventStatus.SENDING,
+        };
+        await addRoomEvent(this, event);
+      };
+      await openAiAssistant();
+
+      await fillIn('[data-test-message-field]', 'Too Large Message');
+      assert.dom('[data-test-message-field]').hasValue('Too Large Message');
```

Comment:
This assertion looks pointless because it doesn't actually test the large message handling. Would this be better suited for a matrix test where you could actually fill in a huge text?